frontend/public/components/overview.jsx

```diff
@@ -0,0 +1,404 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { Button, EmptyState, Icon, Toolbar } from 'patternfly-react';
+import { Helmet } from 'react-helmet';
+
+import { StartGuide } from './start-guide';
+import { FLAGS, connectToFlags, flagPending } from '../features';
+import { Dropdown, Firehose, NavTitle, StatusBox } from './utils';
+import { TextFilter } from './factory';
+import { ProjectOverview } from './project-overview';
+
+const LIST_VIEW = 'list';
+const GRAPH_VIEW = 'graph';
+
+// Stub for future topology graph
+const ApplicationTopology = () =>
+  <div className="co-m-pane__body">
+    <EmptyState>
+      <EmptyState.Title>This view is not yet implemented.</EmptyState.Title>
+    </EmptyState>
+  </div>;
+
+const getOwnedResources = (resources, uid) => {
+  return _.filter(resources, function (resource) {
+    const {ownerReferences} = resource.metadata;
+    return _.some(ownerReferences, {
+      uid,
+      controller: true
+    });
+  });
+};
+
+const OverviewToolbar = ({activeView, groupOptions, handleFilterChange, handleGroupChange, selectedGroup, setView, showViewSelector = false}) =>
+  <Toolbar className="overview-toolbar">
+    <Toolbar.RightContent>
+      <div className="form-group overview-toolbar__form-group">
+        <label style={{marginRight: '10px'}}>
+          Group by label
+        </label>
+        <Dropdown
+          className="overview-toolbar__dropdown"
+          disabled={!_.size(groupOptions)}
+          items={groupOptions}
+          onChange={handleGroupChange}
+          style={{display: 'inline-block'}}
+          title={selectedGroup}
+        />
+      </div>
+      <div className="form-group overview-toolbar__form-group">
+        <TextFilter
+          label="Resources by name"
+          onChange={handleFilterChange}
+        />
+      </div>
+      { showViewSelector &&
```

Comment:
I would take this out of this PR. We can introduce it when we're ready to integrate topology view